arithmetic_analysis/bisection.py

```diff
@@ -30,9 +42,6 @@ def bisection(
         return mid
 
 
-def f(x):
-    return math.pow(x, 3) - 2 * x - 5
-
-
 if __name__ == "__main__":
-    print(bisection(f, 1, 1000))
```

Comment:
Why remove this code?

Reply:
Hmm, I thought I was supposed to move all tests into the doctest. I guess I misunderstood. I can put it back in for sure.